keras2onnx/ke2onnx/simplernn.py

```diff
@@ -48,7 +48,13 @@ def convert_keras_simple_rnn(scope, operator, container):
         rnn_input_names.append('')
 
     # sequence_lens is not able to be converted from input_length
-    rnn_input_names.append('')
+    uses_masking_layer = len(operator.input_masks) == 1
+    if uses_masking_layer:
+        # Mask using sequence_lens input
+        sequence_lengths = scope.get_unique_variable_name(operator.full_name + '_seq_lens')
```

Comment:
where is this tensor fed from the input masking data? do I miss something?

Reply:
The `sequence_lengths` variable defined here, which is passed to the LSTM layer, is used below to connect the `Masking` layer output. 
https://github.com/onnx/keras-onnx/pull/386/files#diff-e1267cefb3f38c599bf90778bb0b4dd1R89